hist/hist/src/TGraph.cxx

```diff
@@ -2152,7 +2152,7 @@ void TGraph::SavePrimitive(std::ostream &out, Option_t *option /*= ""*/)
 
 TString TGraph::SaveArray(std::ostream &out, const char *suffix, Int_t frameNumber, Double_t *arr)
 {
-   const char *name = GetName();
+   const char *name = gInterpreter->MapCppName(GetName());
    if (!name || !*name)
```

Comment:
One need to use `TString name` - if one store output of `MapCppName` method. Like: 
```
TSting name = gInterpreter->MapCppName(GetName());
if (name.IsNull()) name = "Graph";
```

Reply:
I'll check. Thanks